admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -139,7 +139,7 @@ module.exports = {
               loader: 'css-loader',
               options: {
                 minimize: true,
-                sourceMap: true,
+                sourceMap: true
```

Comment:
Why? This is a rule from our eslint config.

Reply:
I changed it